src/dotnet/commands/dotnet-new/FSharp_Console/project.json.template

```diff
@@ -9,31 +10,18 @@
       ]
     }
   },
-  "dependencies": {
-    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160509"
-  },
   "tools": {
-    "dotnet-compile-fsc": {
-      "version": "1.0.0-preview2-*",
-      "imports": [
-        "dnxcore50",
-        "portable-net45+win81",
-        "netstandard1.3"
-      ]
-    }
+    "dotnet-compile-fsc":"1.0.0-preview2-*"
   },
   "frameworks": {
     "netcoreapp1.0": {
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
           "version": "1.0.0"
-        }
-      },
-      "imports": [
-        "portable-net45+win8",
-        "dnxcore50"
-      ]
+        },
+        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629"
```

Comment:
I'd rather keep this explicit.  We don't want newer bits than this build getting pulled down if we publish a new version to NuGet between now and the next release of the SDK.  Although it was stale in the previous release, the fault for that ultimately falls on the us (the F# team) for not making sure we updated the version number correctly to align with the release.